tests/test_path.py

```diff
@@ -173,10 +183,10 @@ def test_nonexistent():
         ("/path/float/True", 422, response_not_valid_float),
         ("/path/float/42", 200, 42),
         ("/path/float/42.5", 200, 42.5),
-        ("/path/bool/foobar", 200, False),
+        ("/path/bool/foobar", 422, response_not_valid_bool),
```

Comment:
pydantic now raises validation errors for not-very-bool-like values

Reply:
Cool, thanks!